### PR TITLE
Repl setup

### PR DESCRIPTION
**Pull request by jpomfret**

### CODE REVIEW

File: blog/content/post/2024/dbatools-repl-setup/index.md [hunk 1]
```diff
@@ -30,14 +55,25 @@ IsPublisher          : False
...
+
+### Setup Publisher
+
+Replication is all about publications, and the articles within those publications. Publications are created on a SQL Server that is setup as a publisher, I'll call `Enable-DbaReplPublishing` to configure `sql1` to be a publisher.
+
+```PowerShell
 Enable-DbaReplPublishing -SqlInstance sql1
+```
 
+The results look very similar to what was returned by `Enable-DbaReplDistributor`, however you can now see the `DistributionDatabase` is populated, and `IsPublisher` has changed to true.
```
Comment (ClaudioESSilva) [inline blog/content/post/2024/dbatools-repl-setup/index.md:74]: missing comma after `however`

File: blog/content/post/2024/dbatools-repl-setup/index.md [hunk 2]
```diff
@@ -47,10 +83,22 @@ DistributionServer   : SQL1
...
+> Throughout this series of blog posts I'm demonstrating the simplest set of parameters for the commands, but there is more available. As you work through setting up replication in your environment, I'd recommend reading the dbatools docs or running `Get-Help <<command>> -Full` in your PowerShell session to see all the available parameters and examples on how to use them.
+
+For the two commands we've just used you can see the dbatools help documentation here:
+
+- [Enable-DbaReplDistributor](https://dbatools.io/Enable-DbaReplDistributor)
+- [Enable-DbaReplPublishing](https://dbatools.io/Enable-DbaReplPublishing)
+
+## Create Publications
 
+As I mentioned the steps are very similar for whatever flavour of replication you want to create, in this post I'll create a transactional replication publication - this is (in my opinion) the most common type that is used in the real world. Transactional replication allows us to replicate changes from one database to another for certain objects in near real time.
```
Comment (ClaudioESSilva) [inline blog/content/post/2024/dbatools-repl-setup/index.md:97]: missing comma: `in this post,`
Also `real time` or `real-time`?

File: blog/content/post/2024/dbatools-repl-setup/index.md [hunk 3]
```diff
@@ -47,10 +83,22 @@ DistributionServer   : SQL1
...
 
-## Add Publication
+The really cool thing here is that this is object level, unlike other technologies like Availability Groups, we can replicate just one table, or even just certain rows from one table based on a filter.
```
Comment (ClaudioESSilva) [inline blog/content/post/2024/dbatools-repl-setup/index.md:99]: I suggest adding `columns` too. Like: `we can replicate just one table, some columns, or even just certain rows based on a filter`

File: blog/content/post/2024/dbatools-repl-setup/index.md [hunk 4]
```diff
@@ -74,34 +124,70 @@ Articles      : {}
...
+{{<
+   figure src="articlefilter.png"
+   alt="Filter Rows page of the testPub publication properties window"
+>}}
+
+More information on this command is available in the dbatools help documentation here:
+
+- [Add-DbaReplArticle](https://dbatools.io/Add-DbaReplArticle)
+
+Alright, the final piece of this puzzle is to add a subscription.
```
Comment (ClaudioESSilva) [inline blog/content/post/2024/dbatools-repl-setup/index.md:182]: Double-check if would make sense to move it to after the next title. Within the `Add a Subscription`

File: blog/content/post/2024/dbatools-repl-setup/index.md [hunk 5]
```diff
@@ -168,6 +241,93 @@ OperatorToEmail        :
...
+
+Now these commands aren't specifically related to replication, but all dbatools commands have this fantastic online help, so feel free to check that out on the website:
+
+- [Get-DbaAgentJob](https://dbatools.io/Get-DbaAgentJob)
+- [Start-DbaAgentJob](https://dbatools.io/Start-DbaAgentJob)
+
+
+## Testing time
+
+With transactional replication, the magic is that rows are replicated (in near real time) as they change on the publisher. Let's prove the replication we've configured is working as expected by inserting a row into the `SalesLT.Customer` table on the `sql1` instance.
```
Comment (ClaudioESSilva) [inline blog/content/post/2024/dbatools-repl-setup/index.md:259]: `real-time`

File: blog/content/post/2024/dbatools-repl-setup/index.md [hunk 6]
```diff
@@ -168,6 +241,93 @@ OperatorToEmail        :
...
+   alt="SSMS showing the select statement results, two rows both for Janet Gates"
+>}}
+
+Let's insert a new row with the following T-SQL.
+
+```sql
+INSERT INTO SalesLT.Customer (NameStyle,Title,FirstName,LastName,PasswordHash,PasswordSalt,rowguid)
+VALUES ('0','Mr.','Bill','Gates','ElzTpSNbUW1Ut+L5cWlfR7MF6nBZia8WpmGaQPjLOJA=','nm7D5e4=','949E9AC8-F8F6-4F7F-8888-87187AC56919')
+```
+
```
Comment (ClaudioESSilva) [inline blog/content/post/2024/dbatools-repl-setup/index.md:285]: Would worth adding the equivalent but using `Invoke-DbaQuery`?

**Review by ClaudioESSilva — CHANGES_REQUESTED** [review]
A couple of suggestions

File: blog/content/post/2024/dbatools-repl-setup/index.md [hunk 7]
```diff
@@ -1,27 +1,52 @@
...
+I'll walk through and demonstrate all the steps to setup replication in this article as dbatools allows us to complete them all. However, I won't go into a lot of details on why or how replication works, or provide guidance on best practices. If you need more information on replication as a technology then I recommend visiting the [Microsoft Docs](https://learn.microsoft.com/en-us/sql/relational-databases/replication/sql-server-replication?view=sql-server-ver16?wt.mc_id=AZ-MVP-5003655).
+
+## Setup Distributor and Publisher
+
+Alright, step 1! 
+
+Replication requires a server that is configured as a distributor, and a server that is configured as a publisher. 
+
+Good news, these pieces of the puzzle can both be configured on the same server which is what I'll demonstrate in my test environment. In environments where replication has a high throughput and\or requires peak performance you can configure a separate server for distribution to move some of the load off of the publisher.
```
[the remark this answers is not in the document]
Comment (jpomfret) [reply blog/content/post/2024/dbatools-repl-setup/index.md:35]: haha ok :)

**Pull request merged**